strings/traumas.json

```
@@ -64,7 +64,8 @@
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
         "DON'T EVER TUCH ME",
         "@pick(semicolon)GIVE ME FREE ROBUX PLEASE JUST ENOUGH FOR SHIRT AND PANTS",
-        "its gonna be like 9/11 all over again but insitaead is gonna be a bitc"
+        "its gonna be like 9/11 all over again but insitaead is gonna be a bitc",
+		"@pick(semicolon)How do I set up the. SHow do I set u p the Singu. how I the scrungulartiy????"
```

```suggestion
        "@pick(semicolon)How do I set up the. SHow do I set u p the Singu. how I the scrungulartiy????"
```

Spaces not tabs here